src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +14,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService service = new UserService();
+        Optional<User> user = Optional.ofNullable(service.findByEmail(email));
```

Comment:
No reason for using Optional here - it makes code more difficult
Optional is useful for method return type (like a warning - we possibly have null inside)